test/basic/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "test-basic",
+  "version": "0.0.0",
+  "private": true,
```

Biome (2.1.2)

[error] The key private was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)